components/tools/OmeroPy/src/omero/rtypes.py

```diff
@@ -294,12 +296,15 @@ def rstring(val):
         return remptystr
     elif isinstance(val, omero.RString):
         return val
-    elif isinstance(val, str):
+    elif isinstance(val, StringTypes):

```

Comment:
I am afraid this will never let you to test ```unicode```. I think you want ```basestring``` that is a base class for ```str``` and ```unicode```.```StringType``` is ```str```.